index.html

```diff
@@ -1,271 +1,282 @@
-
-          <h2 class="project-title">Multi-Post Stories</h2>
-          <p class="project-infor">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
+        </div>
+      </article>
+  
+      <!--fourth project-->
+  
+      <article id="project4" class="project-container">
+  
+        <img class="project-image" src="./images/mobile/Img Placeholder.png" alt="project-image">
+  
+  
+        <div class="description">
+  
+          <h3 class="project-title">Multi-Post Stories</h3>
+          <p class="project-infor"> A daily selection of privately personalized reads no accounts or sign-ups
+            required. has been the industry standard dummy text ever since the 1500s, when an unknown printer took
+            standard dummy text.
           </p>
-
+  
           <ul class="used-tools">
             <li class="project-list">css</li>
-
             <li class="project-list">html</li>
-
             <li class="project-list">bootstrap</li>
-
             <li class="project-list">Ruby</li>
           </ul>
-
           <button type="button" class="see-prj">See Project</button>
-        </article>
-      </section>
-    </main>
-    <footer class="footer">
-      <div class="contact-us-format">
-        <h1 class="contact-us">CONTACT US</h1>
+        </div>
+  </section>
+
+</main>
+
+
+
+ <footer class="footer">
 
-        <p class="offer-help">
+    <section class="contact-section">
+
+      <div class="contact-instructions">
+        <h2 class="contact-us">CONTACT US</h2>
+         
+        <p class="we-help">
           If you have an application you are interested in developing, a feature
           that you need built or a project that needs coding. I’d love to help
           with it
         </p>
       </div>
+  
+      <form action="https://formspree.io/f/xjvlllpk" method="POST" class="form">
 
-     <form  action="https://formspree.io/f/xjvlllpk"  method="POST" class="contact" >
-        <input
-          type="text"
-          id="name"
-          name="name"
-          maxlength="30"
-          placeholder="Text input"
-        required/>
+        <input type="text" class="name1" id="name1" name="name" maxlength="30" placeholder="Text input" required />
 
-        <input class="email" type="email" id="email" name="email" placeholder="jeremy@gmail.com" required/>
+  
+        <input class="email1" type="email" id="email1" name="email" placeholder="jeremy@gmail.com" required />
 
-        <textarea
-          class="message"
-          name="message"
-          id="message"
-          maxlength="500"
-          placeholder="Enter text here"
-          required></textarea>
+  
+        <textarea class="textarea1" name="message" id="textarea1" maxlength="500" placeholder="Enter text here" required></textarea>
```

Comment:
As stated by the previous reviewer, placeholder texts need to be meaningful. Something like `Enter name`, `Enter email` and `Type message here` will suffice. Please fix this.

<img width="393" alt="Screen Shot 2022-06-07 at 12 23 31" src="https://user-images.githubusercontent.com/82426682/172444791-eb688e3b-01a4-4d49-99aa-1d406eeb7b1d.png">